diagnostics-extension/src/index.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/**
+ * @fileoverview Entry-point script of background worker
+ */
+
+console.log('Background worker is up!');
```

Comment:
remove this.

You should listen on install events and do the necessary pre-work (which will be empty for now).
```

self.addEventListener('install', function(event) {
   console.log('Service worker is installed!');
  );
});
```